lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -763,17 +763,17 @@ def test_non_asci_grade_cutoffs(self):
 
     def test_generate_cert_config(self):
         resp = views.progress(self.request, course_id=unicode(self.course.id))
-        self.assertNotContains(resp, 'Create Your Certificate')
+        self.assertNotContains(resp, 'Request Certificate')
 
         # Enable the feature, but do not enable it for this course
         CertificateGenerationConfiguration(enabled=True).save()
         resp = views.progress(self.request, course_id=unicode(self.course.id))
-        self.assertNotContains(resp, 'Create Your Certificate')
+        self.assertNotContains(resp, 'Request Certificate')
 
         # Enable certificate generation for this course
         certs_api.set_cert_generation_enabled(self.course.id, True)
         resp = views.progress(self.request, course_id=unicode(self.course.id))
-        self.assertContains(resp, 'Create Your Certificate')
+        self.assertContains(resp, '')
```

Comment:
This assertion will always be true :)  Maybe `self.assertNotContains(resp, 'Request Certificate')`?